File: lib/common/models/android/auto_selfie_settings.dart
class AndroidAutoSelfieSettings {
  int? textSize;
  bool? counterVisible;
  int? counterTextSize;
  int? manualCropTimeout;
  String? distanceText;
  String? faceNotFoundText;
  String? stableText;
  String? restartText;

  AndroidAutoSelfieSettings(
      {this.textSize,
      this.counterVisible,
      this.counterTextSize,
      this.manualCropTimeout,
      this.distanceText,
      this.faceNotFoundText,
      this.stableText,
      this.restartText});

  Map toJson() {
    return {
      "textSize": textSize,
      "counterVisible": counterVisible,
      "counterTextSize": counterTextSize,
      "manualCropTimeout": manualCropTimeout,
      "distanceText": distanceText,
      "faceNotFoundText": faceNotFoundText,
      "stableText": stableText,
      "restartText": restartText
    };
  }
}

File: android/src/main/kotlin/ai/amani/flutter_amanisdk/modules/config_models/AutoSelfieSettings.kt
package ai.amani.flutter_amanisdk.modules.config_models

import ai.amani.flutter_amanisdk.modules.JSONConvertable
import kotlinx.parcelize.Parcelize

@Parcelize
data class AutoSelfieSettings(val textSize: Int,
                         val counterVisible: Boolean,
                         val counterTextSize: Int,
                         val manualCaptureTimeout: Int,
                         val distanceText: String,
                         val faceNotFoundText: String,
                         val stableText: String,
                         val restartText: String): JSONConvertable